src/components/home/cluster_dashboard_item.js

```diff
@@ -108,6 +108,27 @@ class ClusterDashboardItem extends React.Component {
 
     const hasNodePools = cluster.nodePools && cluster.nodePools.length !== 0;
 
+    if (typeof cluster.delete_date !== 'undefined' && cluster.delete_date) {
+      return (
+        <div className='cluster-dashboard-item well deleted'>
+          <div className='cluster-dashboard-item--label'>
+            <ClusterIDLabel clusterID={cluster.id} copyEnabled />
+          </div>
+
+          <div className='cluster-dashboard-item--content'>
+            <div className='cluster-dashboard-item--title'>
+              <span className='cluster-dashboard-item--name'>
+                {cluster.name}
+              </span>
+            </div>
+            <div className='cluster-dashboard-item--delete-date'>
+              Deleted {relativeDate(cluster.delete_date)}
+            </div>
+          </div>
+        </div>
+      );
+    }
+
```

Comment:
In order to not duplicate code, I suggest you store the `deleted` string in a variable conditionally. Then you can interpollate it wherever you want it to be printed:

```javascript
const deletedClass = typeof cluster.delete_date !== 'undefined' && cluster.delete_date ? 'deleted' : null;

// And  then use it like: 

return (
  <div className={`cluster-dashboard-item well ${deletedClass}`}>
  // ...
)

Reply:
> In order to not duplicate code, I suggest you store the `deleted` string in a variable conditionally. Then you can interpollate it wherever you want it to be printed:
> 
> ```js
> const deletedClass = typeof cluster.delete_date !== 'undefined' && cluster.delete_date ? 'deleted' : null;
> 
> // And  then use it like: 
> 
> return (
>   <div className={`cluster-dashboard-item well ${deletedClass}`}>
>   // ...
> )
> ```

That will actually add the `null` classname if `typeof cluster.delete_date !== 'undefined' && cluster.delete_date` is not true, since you're converting the `deletedClass` variable to a string, and it would have a null value

Reply:
Sorry, true @axbarsan is right, an empty string would do the trick

Reply:
Changed everything to use styled now. Does this comment still apply?